.linkcheck/skip_list.txt

```diff
@@ -13,6 +13,9 @@ https://support.commercetools.com
 https://developers.cloudflare.com
 https://dash.cloudflare.com
 https://www.npmjs.com
+https://accounts.google.com
+https://www.googletagmanager.com
+https://console.cloud.google.com
```

Comment:
Skipping these domains as they end up with too many redirects that don't seem to resolve properly outside of a browser.